builtin/logical/consul/backend_test.go

```diff
@@ -238,14 +249,37 @@ func testBackendRenewRevoke14(t *testing.T, version string) {
 
 	req.Path = "roles/test"
 	req.Data = map[string]interface{}{
-		"policies": []string{"test"},
-		"lease":    "6h",
+		"lease": "6h",
+	}
+	if policiesParam == "both" {
+		req.Data["policies"] = []string{"wrong-name"}
+		req.Data["consul_policies"] = []string{"test"}
+	} else {
+		req.Data[policiesParam] = []string{"test"}
 	}
+
 	_, err = b.HandleRequest(context.Background(), req)
 	if err != nil {
 		t.Fatal(err)
 	}
 
+	read := &logical.Request{
+		Storage:   config.StorageView,
+		Operation: logical.ReadOperation,
+		Path:      "roles/test",
+		Data:      connData,
+	}
+	roleResp, err := b.HandleRequest(context.Background(), read)
+
+	t.Logf("Response consul_policies '%s' should match '[test]'", roleResp.Data["consul_policies"])
```

Comment:
Can we remove this log?

Reply:
I think the majority of the log statements were superfluous to the test cases so I removed most of them including this one.